javascripts/tagmanagerHelper.js

```diff
@@ -266,7 +266,7 @@
         ajaxRequest.send();
     };
     tagManagerHelper.updateDebugSiteFlag = function (url, idContainer, debugFlag) {
-        if (!url || !idContainer || !debugFlag) {
+        if (!url || !idContainer || !debugFlag || url.indexOf('http') !== 0 || url.indexOf('://') < 4) {
```

Comment:
can be simplified to
```suggestion
        if (!url || !idContainer || !debugFlag || /^https?:\/\//.test(url)) {
```